distribution/docs/src/main/resources/_contents/_quickstart/quickstart-configuring-contents.adoc

```diff
@@ -0,0 +1,17 @@
+
+=== Configuring (Quick Install)
+
+Set the configurations needed to run ${branding}
+
+.. Go to ${secure_url}/admin.
+.. Enter the default username of `admin` and the password of `admin`.
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure <<_guest_interceptor,guest claims attributes>> or use defaults.
```

Comment:
That link currently points to an invalid section and I couldn't find any information about guest claim attributes in the current doc. Should we create an empty section in the Securing DDF section so we can link to it here and don't forget to add that information?